script/_nat64

```diff
@@ -151,9 +149,15 @@ nat44_install()
 case "\$1" in
     start)
 EOF
-    for IFNAME in $WLAN_IFNAMES; do
-        echo "      iptables -t nat -A POSTROUTING -o $IFNAME -j MASQUERADE" | sudo tee -a $NAT44_SERVICE
-    done
+    if [ "$NAT64_SERVICE" = tayga ]; then
+        # Although Tayga itself also configures a NAT44 iptables route, this iptables route is used with Tayga
+        # due to some history reason. It might be removed when native NAT64 service is ready.
+        for IFNAME in $WLAN_IFNAMES; do
+            echo "        iptables -t nat -A POSTROUTING -o $IFNAME -j MASQUERADE" | sudo tee -a $NAT44_SERVICE
+        done
+    else
+        echo "        iptables -t nat -A POSTROUTING -s \"$DYNAMIC_POOL\" -j MASQUERADE" | sudo tee -a $NAT44_SERVICE
```

Comment:
Would suggest adding the following rules for NAT64:

```
iptables -t filter -A FORWARD -o $IFNAME -j ACCEPT
iptables -t filter -A FORWARD -i $IFNAME -j ACCEPT
```

Since some platforms will drop all forwarding packets (by iptables) by default